src/components/switch/switch.js

```diff
@@ -35,9 +35,12 @@ class Switch extends Component {
             >
               <input
                 type="radio"
+                disabled={option.disabled}
                 id={option.value}
                 name={name}
-                className={cx(styles.radioInput, theme.radioInput)}
+                className={cx(styles.radioInput, theme.radioInput, {
+                    [theme.disabled]: option.disabled
```

Comment:
is this a style or theme property? in the style is empty and not sure if the theme should take care of this

Reply:
It should be both; fixed it